aptos-move/framework/src/natives/dispatchable_fungible_asset.rs

```diff
@@ -0,0 +1,52 @@
+use super::function_info::extract_function_info;
+use aptos_gas_algebra::InternalGas;
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+use aptos_native_interface::{
+    RawSafeNative, SafeNativeBuilder, SafeNativeContext, SafeNativeError, SafeNativeResult,
+};
+use move_vm_runtime::native_functions::NativeFunction;
+use move_vm_types::{loaded_data::runtime_types::Type, values::Value};
+use smallvec::SmallVec;
+use std::collections::VecDeque;
+
+/***************************************************************************************************
+ * native fun dispatchable_withdraw
+ *
+ *   Directs control flow based on the last argument.
+ *   gas cost: TBD
+ *
+ **************************************************************************************************/
+pub(crate) fn native_dispatch(
+    context: &mut SafeNativeContext,
+    ty_args: Vec<Type>,
+    mut arguments: VecDeque<Value>,
+) -> SafeNativeResult<SmallVec<[Value; 1]>> {
+    let (module_name, func_name) = extract_function_info(&mut arguments)?;
+    if !context.traversal_context().visited.contains_key(&(module_name.address(), module_name.name())) {
+        return Err(SafeNativeError::Abort { abort_code: 4 });
```

Comment:
what does 4 stand for?

Reply:
https://github.com/aptos-labs/aptos-core/blob/a5e14d5301dfc1b304756888d548d3ff7441dcd9/aptos-move/framework/aptos-framework/sources/dispatchable_fungible_asset.move#L30

 This is for the target module is not available in the cache.